### DISCUSSION TITLE
Getting problems when using pubsub

### DISCUSSION THREAD

**Discussion by vendelieu** [redis/lettuce#2190] [General]
I have three applications running on my server:
1 publisher and 2 subscribers, each with their own channel.

After starting the 2nd subscriber stops receiving events at all, maximum receives 1 event once and that's it.

There are thoughts that the 1st subscriber took over the threads and does not give, but I do not know how to check, expand, divide, is it even possible?

**Comment by vendelieu** [comment]
publisher:

```
val redisPubSubClient = RedisClient.create("redis://localhost:6379/1").connectPubSub()
redisPubSubClient.async().publish("channel1", data)
```

subs are of this kind:
```
val redisPubSubClient = RedisClient.create("redis://localhost:6379/1").connectPubSub()
redisPubSubClient.addListener(RedisListener)
redisPubSubClient.async().subscribe("channel1")
```

**Comment by mp911de** [comment]
Can you provide a more complete example? The code looks fine so far, however, the `RedisListener` code could provide a hint for potential issues.
